tests/features/custom_page/collection.custom_page.feature

```diff
@@ -0,0 +1,44 @@
+@api
+Feature: "Add custom page" visibility options.
+  In order to manage custom pages
+  As a collection member
+  I need to be able to add "Custom Page" content through UI.
+  Background:
+    Given users:
+    | name            | mail                  | roles         | pass |
+    | Rufus Drumknott | member@example.com    |               | test |
+    And the following collections:
+    | uri                             | name           | description               | logo     |  owner |
+    | http://joinup.eu/collection/foo | Foo Collection | This is a foo collection. | logo.png |        |
+    | http://joinup.eu/collection/bar | Bar Collection | This is a bar connection. | logo.png |        |
```

Comment:
Owner is empty? Is this a required field? If it is not then this column can be removed. If it is then we have a bug :)